codec/aminojson/internal/testpb/test.pb.go

```diff
@@ -0,0 +1,549 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.28.1
```

Comment:
These should be generated by protoc-gen-go-pulsar, not the official protoc-gen-go.

I think for our use case here it doesn't change much that they are pulsar or official go types, since both respect the protoreflect API. But for consistency I would advise to generate pulsar types only.

See `{core,client/v2,textual}/buf.gen.yaml` for some examples of the `go-pulsar` plugin